xml/System.Text.Encodings.Web/JavaScriptEncoder.xml

```diff
@@ -192,6 +192,8 @@ Unlike the <xref:System.Text.Encodings.Web.JavaScriptEncoder.Default> encoding (
 
 Unlike the <xref:System.Text.Encodings.Web.JavaScriptEncoder.Default> encoder, this encoder instance allows some other characters (such as '+') to go through unescaped and therefore must be used cautiously.
 
+For more information, see [Serialize all characters](/dotnet/standard/serialization/system-text-json-how-to?view=netcore-3.0#serialize-all-characters).
```

Comment:
It looks like the `view=netcore-3.0` is redundant. Correct me if that's wrong.

```suggestion
For more information, see [Serialize all characters](/dotnet/standard/serialization/system-text-json-how-to#serialize-all-characters).
```

Reply:
Good catch, thanks!